specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/ga/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,932 @@
+      "post": {
+        "tags": [
+          "Search"
+        ],
+        "summary": "Purchases the phone number search.",
+        "operationId": "PhoneNumber_PurchasePhoneNumberSearch",
+        "consumes": [
+          "application/json-patch+json",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberPurchaseRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation status using the URL specified in this header.",
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to query for status of the operation",
+                "type": "string"
+              },
+              "Location": {
+                "description": "Url to retrieve the final result after operation completes.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Purchase phone number search": {
+            "$ref": "./examples/PurchaseSearchAsync.json"
+          }
+        }
+      }
+    },
+    "/phoneNumbers/{phoneNumber}/capabilities": {
+      "post": {
+        "tags": [
+          "Update"
+        ],
+        "summary": "Updates the capabilities of a phone number.",
+        "operationId": "PhoneNumber_UpdateCapabilities",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "phoneNumber",
+            "description": "Phone number to be updated. E.g. +11234567890.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Defines the update capabilities request.",
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberCapabilities"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
202 response is missing schema:
"schema": {
  "$ref": "#/definitions/AcquiredPhoneNumber"
}